app/scripts/proactive/templates/job-variable-template.html

```diff
@@ -136,10 +138,13 @@ <h4 class="modal-title"><%- jobName %> </h4>
                 <textarea class="variableValue textareavalues submitVariableValue" rows=1 name="<%- variable.Name %>" data-variable-model="<%- model %>" id="<%- key %>" ><%- variable.Value %></textarea>
                 <label class="short" data-toggle="tooltip" title="<%- model %>"><%- shortenModel %></label>
                 <button title="Browse, Upload User Space files, and Select one folder" value="<%- key %>" class="var-userfolder-button btn far fa-folder-open" type="button" style="background-color:Transparent"></button>
-            <% } else {  %>
-                <textarea class="variableValue textareavalues submitVariableValue" rows=1 name="<%- key %>" data-variable-model="<%- model %>" id="<%- key %>" ><%- variable.Value %></textarea>
-                <label class="Extra" data-toggle="tooltip" title="<%- model %>"><%- shortenModel %></label>
-            <% }  %>
+            <% } else { %>
+                <textarea class="variableValue textareavalues submitVariableValue" rows=1 name="<%- variable.Name %>" data-variable-model="<%- model %>" id="<%- key %>" ><%- variable.Value %></textarea>
+                <label class="short" data-toggle="tooltip" title="<%- model %>"><%- referenceType %></label>
+                <% if( referenceType === "PA:GLOBAL_FILE" ) { %>
```

Comment:
Why only for "PA:GLOBAL_FILE" but not "PA:USER_FILE", "PA:USER_FOLDER", and "PA:GLOBAL_FOLDER" ?

Reply:
It seems to me that dealing with referenceType is similar to mdoel. It could be put in the same place.